.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 *.iml
+*.swp
```

Comment:
I haven't had a .swp file in ages for my vim setup. Most likely because I have this option set `set noswapfile`